app/ui.js

```diff
@@ -180,7 +180,6 @@ var UI;
             UI.initSetting('view_only', false);
             UI.initSetting('path', 'websockify');
             UI.initSetting('repeaterID', '');
-            UI.initSetting('token', '');
```

Comment:
Can't you just change `path` in that case? That's what we do for ThinLinc and it has worked out fine.